app/src/main/java/com/islamey/hamza/wazaker/di/NetworkModule.kt

```diff
@@ -0,0 +1,54 @@
+package com.islamey.hamza.wazaker.di
+
+import com.google.gson.GsonBuilder
+import com.islamey.hamza.wazaker.data.remote.ApiService
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object NetworkModule {
+
+    const val BASE_URL = "https://api.aladhan.com/"
+
+    @Provides
+    @Singleton
+    fun provideOkHttpClient(): OkHttpClient {
+        return OkHttpClient.Builder()
+            .connectTimeout(10, TimeUnit.SECONDS)
+            .readTimeout(10, TimeUnit.SECONDS)
```

Comment:
increase this time, make 30 second as sometimes internets is slow or content is large